database/languages/kotlin.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "name": "Kotlin Tutorial",
+        "description": "This is the official Kotlin tutorial/docs created by Jetbrains. It covers all the topics from beginner to advanced and provides comprehensive documentation on how to effectively implement this programing language in your projects. It also provides a playground where you can learn and practice using Kotlin.",
+        "url": "https://kotlinlang.org/docs/getting-started.html",
+        "category": "languages",
+        "subcategory": "kotlin"
+    },
+    {
+        "name": "Programiz Kotlin",
+        "description": "It is a beginner-friendly tutorial for Kotlin",
```

Comment:
can you please provide the description at 15 words long?

Reply:
I have added a new description